src/drive/web/modules/selection/SelectionBar.jsx

```diff
@@ -19,8 +19,22 @@ const mapDispatchToProps = dispatch => ({
 export default connect(
   mapStateToProps,
   mapDispatchToProps
-)(({ selectionModeActive, ...rest }) => (
-  <div style={{ display: selectionModeActive ? 'inherit' : 'none' }}>
-    <SelectionBar {...rest} />
-  </div>
-))
+)(({ selectionModeActive, actions, ...rest }) => {
+  const convertedActions = DriveActionToSelectionBarAction(actions)
```

Comment:
I think we could use `processSelectionBarActions` here ? At first glance, it is not obvious that this function is processing several action since it is not using the plural form. 2nd it was not clear to me that it could filter the actions, I thought it would only be a transform 1-1 between actions.

I think that if we put the "displayInSelectionBar" property in the action, we can remove this function and do the filtering directly in the selection bar, what do you think ?

Reply:
This is a great idea, but I don't know yet if we really don't want them if the SelectionBar, it's not really clear on either side. I'll try to find out what we want 👍